docs/enrichers.adoc

```diff
@@ -336,3 +336,16 @@ So previous example could be modified to:
 ----
 
 And in case you are running on _Windows_ or _MacOS_, `host`property will be automatically set to the +boot2docker +_ip_.
+
+=== System Properties Injection
+
+Arquillian Cube sets some system properties to be used in tests in case you cannot use enrichers (i.e in case of JUnit rules).
```

Comment:
I meant Arquillian Enrichers. If you are using Arquillian Cube or Arquillian APE JUnit rules then you are not using  Arquillian runner.